demo/api.js

```diff
@@ -37,15 +44,14 @@ getApp({
 });
 
 require('@babel/register'); // eslint-disable-line  import/no-extraneous-dependencies
-const {
-	editController,
-	viewController,
-	deleteController,
-	anotherController,
-} = require('./cms');
+const { editController, viewController, deleteController } = require('./cms');
 
 const parseBody = bodyParser.urlencoded({ limit: '8mb', extended: true });
-app.get('/lalala', anotherController);
+app.get('/', (req, res) => {
```

Comment:
If you want to code golf it then this is just fine, the browser can error correct the rest!

```<a href="/graphiql">GraphQL explorer.</a>```

Reply:
yeah - I was originally gonna add a lot more links but then decided against. Gotta love the 'most hostile development environment on the planet's ability to deal with bad input